TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/CameraTest.java

```diff
@@ -0,0 +1,31 @@
+package org.firstinspires.ftc.teamcode.opmodes;
+
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.teamcode.hardware.VisionCamera;
+import org.firstinspires.ftc.teamcode.utility.FieldSide;
+
+@TeleOp(name = "CameraTest")
+public class CameraTest extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        // ALWAYS SET TELEMETRY AUTOCLEAR to TRUE!!!
```

Comment:
Could we add an note to these indicating _why_ we must always set telemetry autoclear to TRUE?